### DISCUSSION TITLE
Multiple Github Accounts on Github Layout

### DISCUSSION THREAD

**Discussion by davesaah** [Ice-Hazymoon/hugo-theme-luna#65] [Ideas]
Hello @Ice-Hazymoon 👋🏼 

Thank for you for the amazing work you've done with hugo-luna-theme. Wanted to asked if you have considered allowing multiple github connections for the github page layout. This could prove useful for people who woud want to show the personal github profile as well as their organisations.

Thanks 🙂

**Comment by Ice-Hazymoon** [comment]
It's a great idea, but I don't have such a plan yet, because only few people have such a need, and you can try to do it yourself. This theme was created for individual users, for organizations or teams, you can try to look at other themes.

Thank you for using luna theme 😉

**Comment by Ice-Hazymoon** [comment]
If you want to implement this feature yourself, you can ask questions in the discussion forum if you encounter any problems, and I will try to answer them for you.

**Reply by davesaah** [reply]
Sure, thank you very much
